### PR TITLE
challenge update

### PR DESCRIPTION
**Pull request by ebbauer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,38 +8,28 @@ Yawoen company has hired you to implement a Data API.
 
 **1 - Load company data in a database**
 
-Read data from CSV file and load into the database to create a catalog named **companies**.
+Read data from CSV file and load into the database to create an entity named **companies**.
 
 This catalog should contain the following fields: id, company name and zip code.
```
Comment (kamilash) [inline README.md]: "catalog" name is duplicated here, needs change too.

File: README.md [hunk 2]
```diff
@@ -8,38 +8,28 @@ Yawoen company has hired you to implement a Data API.
...
     - Website: string
 
-Example: 
-
-```
-{
-	"name": "Yawoen Business Solutions",
-	"zip":"10023",
-	"website": "www.yawoen.com"
-}
```
Comment (kamilash) [inline README.md:39]: Not happy with the example?

File: README.md (hunk 1 above)
Comment (ebbauer) [reply README.md]: thanks!

File: README.md (hunk 2 above)
Comment (ebbauer) [reply README.md:39]: it is an output example and we are talking about the input

**Review by matheusvill — APPROVED** [review]
Nice! :+1:

**Review by kamilash — APPROVED** [review]
LGTM! :)

**Pull request merged**